recipe/meta.yaml

```diff
@@ -1,83 +1,92 @@
-{% set cudnn_cuda_version = cuda_compiler_version %}
+{% set cudnn_cuda_version = "11.5" %}
 {% endif %}
 
 package:
   name: cudnn
   version: {{ version }}
 
 source:
-  url: https://developer.download.nvidia.com/compute/redist/cudnn/v{{ ".".join(version.split(".")[:3]) }}/cudnn-{{ cudnn_cuda_version }}-linux-x64-v{{ version }}.tgz  # [linux64]
-  url: https://developer.download.nvidia.com/compute/redist/cudnn/v{{ ".".join(version.split(".")[:3]) }}/cudnn-{{ cudnn_cuda_version }}-linux-ppc64le-v{{ version }}.tgz  # [ppc64le]
-  url: https://developer.download.nvidia.com/compute/redist/cudnn/v{{ ".".join(version.split(".")[:3]) }}/cudnn-{{ cudnn_cuda_version }}-windows10-x64-v{{ version }}.zip  # [win and cuda_compiler_version == "10.2"]
-  url: https://developer.download.nvidia.com/compute/redist/cudnn/v{{ ".".join(version.split(".")[:3]) }}/cudnn-{{ cudnn_cuda_version }}-windows-x64-v{{ version }}.zip  # [win and cuda_compiler_version == "11.2"]
-  sha256: 3e70876bdcf44f856d9c9dbdbde07ceec43005e97d5ffe83338b255c8466151f  # [win64 and cuda_compiler_version == "10.2"]
-  sha256: 5b9bf2dc4670fb1519ef55e13da5123f0b6b39fac5e6138e31388b269808d5f2  # [win64 and cuda_compiler_version == "11.2"]
-  sha256: fd6321ff3bce4ce0cb3342e5bd38c96dcf3b073d44d0808962711c518b6d61e2  # [linux64 and cuda_compiler_version == "10.2"]
-  sha256: 39412acd9ef5dd27954b6b9f5df75bd381c5d7ceb7979af6c743a7f4521f9c77  # [linux64 and cuda_compiler_version == "11.2"]
+  - url: https://developer.download.nvidia.com/compute/redist/cudnn/v{{ ".".join(version.split(".")[:3]) }}/local_installers/{{ cudnn_cuda_version }}/cudnn-linux-x86_64-{{ version }}_cuda{{ cudnn_cuda_version }}-archive.tar.xz  # [linux64]
+    sha256: 5500953c08c5e5d1dddcfda234f9efbddcdbe43a53b26dc0a82c723fa170c457  # [linux64 and cuda_compiler_version == "11.2"]
+  - url: https://developer.download.nvidia.com/compute/redist/cudnn/v{{ ".".join(version.split(".")[:3]) }}/local_installers/{{ cudnn_cuda_version }}/cudnn-linux-ppc64le-{{ version }}_cuda{{ cudnn_cuda_version }}-archive.tar.xz  # [ppc64le]
+    sha256: 0581bce48023a3ee71c3a819aaefcabe693eca18b61e2521dc5f8e6e71567b1b  # [ppc64le and cuda_compiler_version == "11.2"]
+  - url: https://developer.download.nvidia.com/compute/redist/cudnn/v{{ ".".join(version.split(".")[:3]) }}/local_installers/{{ cudnn_cuda_version }}/cudnn-linux-sbsa-{{ version }}_cuda{{ cudnn_cuda_version }}-archive.tar.xz  # [aarch64]
+    sha256: 7eb8c96bfeec98e8aa7cea1e95633d2a9481fc99040eb0311d31bf137a7aa6ea  # [aarch64 and cuda_compiler_version == "11.2"]
+  - url: https://developer.download.nvidia.com/compute/redist/cudnn/v{{ ".".join(version.split(".")[:3]) }}/local_installers/{{ cudnn_cuda_version }}/cudnn-windows-x86_64-{{ version }}_cuda{{ cudnn_cuda_version }}-archive.zip  # [win64]
+    sha256: 9e36eef803f1cf9ab24846dc133a3014fdc548775ee29073e8466d415957a1c0  # [win64 and cuda_compiler_version == "11.2"]
+  - url: http://www.winimage.com/zLibDll/zlib123dllx64.zip  # [win64]
```

Comment:
The [installation instructions suggest](https://docs.nvidia.com/deeplearning/cudnn/install-guide/index.html#install-zlib-windows) using this dll from the URL.
`zlib` feedstock uses a different name for the DLL - `zlib.dll`, so I think it's fine to redistribute `zLibDll/zlib123dllx64.zip` here as part of the cuDNN package.

Reply:
Ideally we should build it ourselves, either as a new feedstock or a different output here.

If that fails and vendoring is the only option, the License must be included in the meta.yaml.

Reply:
@IvanYashchuk do you mind taking over and finishing the last mile for me? 🙂

Reply:
Yes, I will finish that.